src-built-in/components/toolbar/src/app.jsx

```diff
@@ -12,20 +12,27 @@ import React from "react";
 import { ToolbarShell } from "@chartiq/finsemble-ui/lib/components/toolbar/ToolbarShell";
 import { onReady } from "@chartiq/finsemble-ui/lib/effects/onReadyEffects";
 import { DragHandle } from '@chartiq/finsemble-ui/lib/components/toolbar/DragHandle'
-import { DummyExport } from '@chartiq/finsemble-ui/lib/components/dummyExport'
+import { RevealAll } from '@chartiq/finsemble-ui/lib/components/toolbar/revealAll';
 import '../toolbar.css';
 const Toolbar = () => {
 	return (
 		<ToolbarShell>
-			<DragHandle/>
-			<DummyExport/>
-			{/* Drag Handle */}
-			{/* Workspace Management Menu */}
-			{/* App Menu */}
-			{/* Favorites Section */}
-			{/* Minimize All */}
-			{/* Reveal All */}
-			{/* Auto Arrange */}
+			<div className="finsemble-toolbar-section left">
```

Comment:
Should we do `div`'s like this or have a specific component like
```jsx
import { ToolbarSection } from "@chartiq/finsemble-ui/components
...
<ToolbarShell>
   <ToolbarSection position=left>
    ...
   </ToolbarSection>
   <ToolbarSection position=center>
    ...
   </ToolbarSection>
   <ToolbarSection position=right>
    ...
   </ToolbarSection>
</ToolbarShell>
```

I can see pro's and con's to both.

Reply:
I like just doing the div - it gives our client developers more options to customize

Reply:
This was meant to be a shortterm solution, we need to decide as a team which we want.

Reply:
Terry decided Meredith's current way is best.